angular-ui/src/app/navbar/navbar.component.ts

```diff
@@ -1,29 +1,43 @@
 import { Component, OnInit } from '@angular/core';
 import { UserProfileComponent } from '../profile/edit-user-profile/edit-user-profile.component';
 import { AuthService } from '../user/auth.service';
+import { User } from '../user/user';
 @Component({
   selector: 'app-navbar',
   templateUrl: './navbar.component.html',
   styleUrls: ['./navbar.component.css'],
 })
 
-//get the logged in variable data into here somehow .. 
 export class NavbarComponent implements OnInit {
   isLoggedIn: boolean;
+  currentUser: User | null;
 
+<<<<<<< HEAD
```

Comment:
I think this is the last one that is strange. but keep lines 20-23. delete 15-19 and 24.